templates/prometheusproxynetworkpolicy.go

```diff
@@ -0,0 +1,56 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+var PrometheusProxyPort = intstr.FromInt(9339)
+var PrometheusProxyProtocol = corev1.ProtocolTCP
```

Comment:
grouping + local scoping 
```suggestion
var (
    prometheusProxyPort = intstr.FromInt(9339)
    prometheusProxyProtocol = corev1.ProtocolTCP
)
```

Reply:
Done in latest commit